Sources/TestsCommon/CommonTestUtils.swift

```diff
@@ -216,6 +216,13 @@ public struct TestRequirement: Decodable {
         )
     ]
 
+    public static let blockTimeSupport: [TestRequirement] = [
+        TestRequirement(
+            minServerVersion: ServerVersion.mongodBlockTimeSupport,
+            acceptableTopologies: [.single, .replicaSet]
```

Comment:
This option should be fine to work on any topology with that version, so I think we can just omit `acceptableTopologies` here.

Reply:
done